selfdrive/ui/qt/home.cc

```diff
@@ -284,7 +284,9 @@ void GLWindow::timerUpdate() {
   handle_display_state(&ui_state, false);
 
   ui_update(&ui_state);
-  repaint();
+  if (ui_state.vipc_client->connected) {
```

Comment:
hmm, this wouldn't update the sidebar at all while offroad

Reply:
Yes it is. This is indeed a problem